compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -453,7 +460,12 @@ def cmd_start_endpoint(
             startup_proc_title = f"Endpoint starting up for {uname} [{args_title}]"
             setproctitle.setproctitle(startup_proc_title)
 
-            amqp_creds = json.dumps(kwargs.get("amqp_creds"))
+            stdin_data_dict = {
+                "amqp_creds": kwargs.get("amqp_creds"),
+                "config": render_config_user_template(self.conf_dir, user_opts),
+            }
+            stdin_data = json.dumps(stdin_data_dict)
+            exit_code += 1
```

Comment:
What is our largest YAML configuration example we have?  I want to make sure that this methodology is robust to whatever we're likely to send down the pipe: I'm now concerned that by sending the `config` in addition to the credentials that we may be close to blocking when we fill the kernel's pipe buffer.

Can talk out of band on this detail if that would be more expedient.

Reply:
Discussed separately: the best path forward is to dynamically check the local system's max pipe buffer size and validate that the data we are sending is smaller. If it isn't, we'll raise an exception with a useful error message.